main.go

```diff
@@ -244,11 +211,32 @@ func main() {
 		}
 
 	case "create-build", "create-build <path>":
+		var androidManifestPath string
+		var BaseOptions build.CreateBuildInfo
 
-		if commands.ApiKey == "" {
-			log.Error("no API key provided", 1)
+		UserBuildOptions := build.PopulateFromCliOpts(commands)
+
+		BaseOptions = build.PopulateFromPath(commands.CreateBuild.Path[0])
+
+		if commands.CreateBuild.AndroidAab != "" {
+			androidManifestPath, err = android.GetAndroidManifestFileFromAAB(string(commands.CreateBuild.AndroidAab))
+
+			if err != nil {
+				log.Error(err.Error(), 1)
+			}
+		}
+
+		if androidManifestPath == "" {
+			androidManifestPath = string(commands.CreateBuild.AppManifest)
 		}
 
+		if androidManifestPath != "" {
+			ManifestBuildOptions := build.PopulateFromAndroidManifest(androidManifestPath)
+			BaseOptions = build.TheGreatMerge(BaseOptions, ManifestBuildOptions)
+		}
+
+		FinalMerge := build.TheGreatMerge(UserBuildOptions, BaseOptions)
```

Comment:
Can all this go into a func in `build`? So that all that's left on the top level is:

1. Create build info
2. Validate build info
3. Get the endpoint URL
4. Call the process function